client/src/components/notifications/MultipleUsersJoinPopover.vue

```diff
@@ -0,0 +1,106 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <ion-list class="users-list">
+    <ion-item
+      class="users-list-item body"
+      button
+      lines="none"
+      v-for="user in userRoles"
+      :key="user.info.id"
+    >
+      <div class="users-list-item-content">
+        <user-avatar-name
+          :user-avatar="userRoles ? user.info.humanHandle.label : ''"
```

Comment:
You shouldn't need the check, simply do `:user-avatar=user.info.humanHandle.label`